semantic-conventions/CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+# Semantic Conventions build tools
+
+This document provides `semantic-convention` specific information for contributions.
+
+Read OpenTelemetry project [contributing
+guide](https://github.com/open-telemetry/community/blob/master/CONTRIBUTING.md)
+for general information about the project.
+
+## Development setup
+
+### Prerequisites
+
+- Python
+- Docker (optional)
+
+### Building
+
+Any changes to the build process should be reflected in this repositiory's `.github/workflows/semconvgen.yml` workflow, in addition to this document and the [CHANGELOG](CHANGELOG.md).
+
+For local development, it may be helpful to use an isolated environment such as [venv](https://docs.python.org/3/library/venv.html), or whatever your preferred IDE may offer. Doing so will avoid polluting your system's path with dependencies specific to this project, and allow for more reproducible builds. For detailed instructions setting up python, see [venv setup](#venv-setup)
+
+Ensure dependencies are installed and on your `PATH` before building. If you're using a terminal, you can install your dependencies by running
+
+```bash
+pip install --upgrade pip
+pip install --upgrade setuptools wheel
+pip install --requirement dev-requirements.txt
+pip install --upgrade --editable .
+```
+
+### Deployment
```

Comment:
Not sure if "deployment" is the right heading here. Deploying (to a docker registry) only happens inside the actual github build, locally you will usually just build a docker image.

Reply:
"building a local docker image"

Reply:
Or just "Building Docker image"